tests/test-backend-ops.cpp

```diff
@@ -2666,6 +2691,51 @@ struct test_cross_entropy_loss : public test_case {
     }
 };
 
+// GGML_OP_OPT_STEP_ADAMW
+struct test_opt_step_adamw : public test_case {
+    const ggml_type type;
+    const std::array<int64_t, 4> ne;
+    const float alpha;
+    const float beta1;
+    const float beta2;
+    const float eps;
+    const float wd;
+
+    std::string vars() override {
+        return VARS_TO_STR2(type, ne);
```

Comment:
All the parameters relevant to the test need to be returned here so that we know what is being tested.

Reply:
Thanks, I forgot.